app/controllers/characters_controller.rb

```diff
@@ -1,9 +1,11 @@
 class CharactersController < ApplicationController
-  before_action :user_logged_in?
+  def show
+    render layout: 'wide',  :locals => {:background => "doom"}
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.